scripts/release-utils.js

```diff
@@ -94,8 +96,44 @@ function publishAndroidArtifactsToMaven(releaseVersion, isNightly) {
   echo('Published artifacts to Maven Central');
 }
 
+function generateiOSArtifacts(
+  jsiFolder,
+  hermesUtilsFolder,
+  hermesCoreSourceFolder,
+  buildType,
+  releaseVersion,
+  targetFolder,
+) {
+  pushd(`${hermesCoreSourceFolder}`);
+
+  //Generating iOS Artifacts
+  exec(
+    `JSI_PATH=${jsiFolder} BUILD_TYPE=${buildType} ${hermesUtilsFolder}/build-mac-framework.sh`,
+  );
+
+  exec(
+    `JSI_PATH=${jsiFolder} BUILD_TYPE=${buildType} ${hermesUtilsFolder}/build-ios-framework.sh`,
+  );
+
+  exec(
+    `JSI_PATH=${jsiFolder} BUILD_TYPE=${buildType} ${hermesUtilsFolder}/build-apple-framework.sh`,
```

Comment:
Do we really need this? IIRC `build-apple-framework.sh` doesn't have any executable top-level code, and is sourced  by both `build-mac-framework.sh` and `build-ios-framework.sh`.

Reply:
Goood question - I just did what @cipolleschi and @hramos do in the podspec; I wanted to make sure to be as close as possible to have the "end code" artifacts as similar as possible to what the end user will experience with a release. So I'll leave it up to them to give the last word on what's needed here :)

Reply:
I think this line can go completely. I think that @kelset was referring to the CI more than the podspec.
In the CI, the `build-apple-framework` is only `source`d to get access to the `get_release_version` function (See [these lines](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1230-L1234)), so we can safely remove this call.